src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTester.java

```diff
@@ -42,16 +42,17 @@
 public class UmiAwareMarkDuplicatesWithMateCigarTester extends AbstractMarkDuplicatesCommandLineProgramTester {
     private int readNameCounter = 0;
     private List<String> expectedAssignedUmis;
-    private final boolean allowMissingUmis;
 
     // This tag is only used for testing, it indicates what we expect to see in the inferred UMI tag.
     private final String expectedUmiTag = "RE";
 
-    UmiAwareMarkDuplicatesWithMateCigarTester(final boolean allowMissingUmis) {
```

Comment:
I do use this parameter.  I've added a comment to clarify this section.
This was done so that I could inherit tests.